addons/web_editor/static/image_shapes/devices/imac_3d_01.svg

```diff
@@ -0,0 +1,76 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xhtml="http://www.w3.org/1999/xhtml" viewBox="0 0 800 952" data-forced-size="true" width="800" height="952" data-img-aspect-ratio="16:9" data-img-perspective="[[5.06, 3.42], [92.25, 18.6], [97.11, 67.29], [9.92, 67.81]]">
+    <defs>
+        <linearGradient id="gradient_01" x1="435.7" y1="1006.24" x2="410.78" y2="794.45" gradientUnits="userSpaceOnUse">
```

Comment:
I still think there is something wrong about the perspective of the imacs and I am still not sure about the green color used by default, what do you all think @bvr-odoo @Brieuc-brd ?

Reply:
I would use a gray in any case as the green could be any colors depending theme colors.
For the perspective, I agree that it could be improved (maybe if the screen is a little less rotated and more frontal)

Reply:
Hello @qsm-odoo @bvr-odoo,
Ok I'll update the perspective of this shape 👍. 
About the color I'm not sure to set the gray by default. Color is one of the main feature of this device (Apple is clearly putting it forward : [https://www.apple.com/imac-24/](https://www.apple.com/imac-24/)). To me it's logical to let the primary color. What do you think ?
<img width="1442" alt="Capture d’écran 2022-01-03 à 09 34 34" src="https://user-images.githubusercontent.com/80679690/147912671-82e98d02-3682-4cfe-8846-33656e3896f9.png">

Reply:
Even if 7 colors are available, I would not let the screen be randomly generated with an unavailable color (brown for example) ... I put it "silver" by default for all themes. But that's just my opinion;)

Reply:
The iMac shape is updated 😉
![Capture-d’écran-2022-01-03-à-13 26 56](https://user-images.githubusercontent.com/80679690/147936486-032f0f1b-dd2e-4b38-b80d-7e20609ad81a.jpg)